Makefile

```diff
@@ -2,6 +2,11 @@ CLANG_FORMAT ?= clang-format
 
 .PHONY: database format clean
 
+env:
+	virtualenv --python=python3 env
+	. env/bin/activate; pip install -r requirements.txt
+	ln -s $(PWD)/prjxray env/lib/python3.*/site-packages/
```

Comment:
Change to `ln -sf` to make this make rule idempotent.

Reply:
Done.